### ISSUE TITLE
Suspicious line in `utils.compute()`

### ISSUE THREAD

**Issue by indiscripts** [Pomax/bezierjs#202]
The `compute()` function of `utils.js` contains an [assignement](https://github.com/Pomax/bezierjs/blob/cb454740a5f79823f6a778cf81df7680ad5ea8a3/src/utils.js#L165) that seems wrong:

~~~~
// . . .
if (typeof dCpts[i].z !== "undefined") {
          dCpts[i] = dCpts[i].z + (dCpts[i + 1].z - dCpts[i].z) * t;
        }
// . . .
~~~~

IMHO it should be replaced by:

~~~~
// . . .
if (typeof dCpts[i].z !== "undefined") {
          dCpts[i].z = dCpts[i].z + (dCpts[i + 1].z - dCpts[i].z) * t;
        }
// . . .
~~~~

**Comment by Pomax** [comment]
Well spotted.

**Comment by Pomax** [comment]
Fixed in 6.1.4
